ui/pages/auth.html

```diff
@@ -51,9 +52,51 @@ <h2>SIGN UP</h2>
 							</div>
 						</div>
 						<div class="to-center" id="form-link">
-							<span>Already a member?</span><a href="#">Sign In</a>
+							<span>Already a member?</span><a onclick="flip()" href="#">Sign In</a>
```

Comment:
@Nziranziza We are not supposed to use javascript for the mockup, just stick to html and css